roles/openshift_version/meta/main.yml

```diff
@@ -12,6 +12,7 @@ galaxy_info:
   categories:
   - cloud
 dependencies:
+- role: openshift_inventory
```

Comment:
Why both `openshift_version` and `openshift_repos` depend on the new `openshift_inventory`?

BTW, maybe naming it `openshift_inventory_validation` would be more specific? So that we avoid turning it into other things like a dynamic inventory role or anything of the like.